### DISCUSSION TITLE
%E4%B8%8D%E7%9F%A5%E9%81%93%E5%A4%9A%E5%B0%91%E4%B8%8D%E6%98%AF%E7%A8%8B%E5%BA%8F%E5%91%98%E7%9A%84%E4%BA%BA%E5%9C%A8%E7%9E%8E%E5%87%A0%E6%8A%8A%E6%B0%B4%E5%91%A2

### DISCUSSION THREAD

**Discussion by daxiazilong** [WeNeedHome/SummaryOfLoanSuspension#854] [Q&A] [answered]
%E4%B8%8D%E7%9F%A5%E6%89%80%E4%BA%91

**Comment by k0spider** [comment]
%E4%B8%8A%E7%8F%AD%E6%91%B8%E9%B1%BC%E7%9A%84%E6%97%B6%E5%80%99%E6%83%B3%E5%85%B3%E6%B3%A8%E7%9A%84%EF%BC%8C%E7%BB%93%E6%9E%9C%E6%84%9F%E8%A7%89%E8%BF%99%E5%87%A0%E5%A4%A9%E7%83%AD%E5%BA%A6%E5%B7%B2%E7%BB%8F%E6%B2%A1%E4%BA%86%EF%BC%8C%E7%BB%88%E7%A9%B6%E8%BF%98%E6%98%AF%E6%95%8C%E4%B8%8D%E8%BF%87%E4%BB%96%E4%BB%AC%E3%80%82

**Reply by daxionga** [reply]
%E6%88%91%E5%BE%88%E5%A5%87%E6%80%AA%EF%BC%8C%E8%BF%99%E4%B8%AA%E9%A1%B9%E7%9B%AE%E8%BF%99%E4%B9%88%E7%81%AB%E6%80%8E%E4%B9%88%E9%83%BD%E6%B2%A1%E4%B8%8Agithub%20trending%EF%BC%8C%E6%98%AF%E6%88%91%E7%9A%84%E8%AE%A4%E7%9F%A5%E6%9C%89%E9%97%AE%E9%A2%98%E5%90%97%EF%BC%8C%E9%82%A3%E5%BE%97%E5%A4%9A%E7%81%AB%E6%89%8D%E8%83%BD%E4%B8%8A%E6%A6%9C

**Comment by lhxone** [comment]
%E7%A8%8B%E5%BA%8F%E5%91%98%E7%9A%84%E5%9F%BA%E7%A1%80%E4%BF%AE%E5%85%BB%E4%B9%8B%E5%8A%A0%E5%AF%86%E9%80%9A%E8%AF%9D

**Comment by 2008zhangguoli** [comment]
%E7%8E%B0%E5%9C%A8%E6%80%8E%E4%B9%88%E9%83%BD%E7%94%A8%E7%BC%96%E7%A0%81%E7%95%99%E8%A8%80%E4%BA%86%E5%95%8A

**Reply by daxiazilong** [reply]
%E5%92%8C%E9%82%A3%E4%BA%9B%E5%8F%AA%E5%BB%BA%E5%8F%B7%E4%BA%86%E4%B8%80%E4%B8%A4%E5%A4%A9%E7%9A%84%E4%BA%BA%E5%AF%B9%E8%AF%9D%20%E5%BF%83%E5%A4%AA%E7%B4%AF%20%E6%A0%B9%E6%9C%AC%E4%B8%8D%E7%9F%A5%E9%81%93%E8%AF%B4tm%E4%BB%80%E4%B9%88

**Reply by ztex** [reply]
%E7%9C%8B%E4%B9%90%E5%AD%90%E4%B8%8D%E5%B0%B1%E8%A1%8C%E4%BA%86%E5%90%97%EF%BC%8C%E5%B4%A9%E6%BA%83%E4%B8%8D%E5%B4%A9%E6%BA%83%E8%83%BD%E6%80%8E%E4%B9%88%E6%A0%B7%E5%91%A2%EF%BC%8C%E4%B8%8D%E9%87%8D%E8%A6%81

**Comment by boweic** [comment]
%E4%B8%8D%E7%9F%A5%E9%81%93%EF%BC%8C%E5%8F%8D%E6%AD%A3%E6%88%91%E4%B8%8D%E6%98%AF%E7%A8%8B%E5%BA%8F%E5%91%98

**Comment by oceanzhang860411** [comment]
%e6%9c%ac%e6%9d%a5%e5%b0%b1%e4%b8%8d%e6%98%af%e5%9b%bd%e5%86%85%ef%bc%8c%e8%af%b4%e4%bb%80%e4%b9%88%e9%83%bd%e5%8f%af%e4%bb%a5%ef%bc%8c%e5%90%a6%e5%88%99%e4%b8%8d%e5%a6%82%e5%9b%9e%e5%a2%99%e5%86%85

**Comment by iosyyy** [comment]
%E8%83%BD%E4%B8%8Agithub%E7%9A%84%E5%A4%A7%E5%A4%9A%E9%83%BD%E6%98%AF%E7%A8%8B%E5%BA%8F%E5%91%98%E6%8A%8A

**Comment by KHW2099** [comment]
%E5%B0%91%E7%82%B9%E4%B8%AA%E4%BA%BA%E5%88%A9%E7%9B%8A%EF%BC%8C%E6%9C%AC%E6%9D%A5%E5%B0%B1%E6%98%AF%E4%B8%AA%E5%A5%91%E6%9C%BA%EF%BC%8C%E5%9B%A2%E7%BB%93%E8%B5%B7%E6%9D%A5%E5%B0%B1%E9%82%A3%E4%B9%88%E9%9A%BE%E5%90%97%EF%BC%9F

**Reply by jbl19860422** [reply]
%E5%8F%AA%E8%83%BD%E8%AF%B4%E5%BE%88%E9%9A%BE%EF%BC%8C%E5%9B%BD%E5%86%85%E6%B2%A1%E6%9C%89%E6%B8%B8%E8%A1%8C%E6%9D%83%E5%8A%9B%EF%BC%8C%E6%B2%A1%E6%9C%89%E5%8F%91%E8%A8%80%E6%9D%83%E5%8A%9B%EF%BC%8C%E4%BD%A0%E7%9C%8B%E7%9C%8B%E8%80%81%E5%A4%96%EF%BC%8C%E5%8A%A8%E4%B8%8D%E5%8A%A8%E5%B0%B1%E6%B8%B8%E8%A1%8C%E7%A4%BA%E5%A8%81%EF%BC%8C%E5%8A%A8%E4%B8%8D%E5%8A%A8%E5%B0%B1%E5%BE%80%E6%80%BB%E7%BB%9F%E8%BA%AB%E4%B8%8A%E6%89%94%E9%B8%A1%E8%9B%8B%EF%BC%8C%E6%81%B6%E6%90%9E%E6%80%BB%E7%BB%9F%E9%83%BD%E6%98%AF%E5%B0%8F%E4%BA%8B%E3%80%82%E5%9B%BD%E5%86%85%E5%B0%B1%E6%98%AF%E7%A5%9E%EF%BC%8C%E7%A2%B0%E4%B8%8D%E5%BE%97%E7%9A%84%E7%A5%9E%E3%80%82%0A%E8%BF%99%E7%A7%8D%E6%83%85%E5%86%B5%EF%BC%8C%E4%B8%8D%E6%98%AF%E4%B8%80%E4%B8%AA%E4%BA%BA%E8%83%BD%E6%89%AD%E8%BD%AC%E7%9A%84%EF%BC%8C%E9%99%A4%E9%9D%9E%E4%BD%A0%E6%98%AF%E5%9B%BD%E5%AE%B6%E9%A2%86%E5%AF%BC%E4%BA%BA%EF%BC%8C%E7%84%B6%E5%90%8E%E6%B4%BB%E5%BE%97%E4%BA%86%E6%9D%83%E5%8A%9B%EF%BC%8C%E7%84%B6%E5%90%8E%E4%BD%A0%E6%80%9D%E6%83%B3%E5%BC%80%E6%98%8E%EF%BC%8C%E8%A7%89%E5%BE%97%E5%BA%94%E8%AF%A5%E4%BA%BA%E4%BA%BA%E5%B9%B3%E7%AD%89%EF%BC%8C%E5%90%A6%E5%88%99%E6%97%A2%E5%BE%97%E5%88%A9%E7%9B%8A%E8%80%85%EF%BC%8C%E4%B8%8D%E4%BC%9A%E6%84%BF%E6%84%8F%E5%88%86%E4%BA%AB%E6%9D%83%E5%8A%9B%E7%BB%99%E4%BA%BA%E6%B0%91%EF%BC%8C%E4%BD%A0%E8%BF%9E%E4%BA%BA%E5%A4%A7%E4%BB%A3%E8%A1%A8%E6%98%AF%E6%80%8E%E4%B9%88%E9%80%89%E7%9A%84%E9%83%BD%E4%B8%8D%E7%9F%A5%E9%81%93%EF%BC%8C%E9%82%A3%E4%BA%9B%E4%BA%BA%E5%A4%A7%E4%BB%A3%E8%A1%A8%E8%BF%9E%E8%87%AA%E5%B7%B1%E6%98%AF%E6%80%8E%E4%B9%88%E8%A2%AB%E9%80%89%E4%B8%8A%E7%9A%84%E4%B9%9F%E4%B8%80%E6%97%A0%E6%89%80%E7%9F%A5%EF%BC%8C%E4%B8%80%E4%B8%AA%E4%B8%AA%E8%82%A5%E5%A4%B4%E5%A4%A7%E8%80%B3%EF%BC%8C%E8%AF%B4%E8%AF%9D%E5%86%A0%E5%86%95%E5%A0%82%E7%9A%87%EF%BC%8C%E5%8F%82%E5%8A%A0%E4%B8%A4%E4%BC%9A%EF%BC%8C%E6%97%A0%E9%9D%9E%E6%98%AF%E5%BD%B0%E6%98%BE%E8%87%AA%E5%B7%B1%E6%B4%BB%E5%BE%97%E6%9D%83%E5%8A%9B%E7%9A%84%E8%8D%A3%E8%80%80%EF%BC%8C%E5%B9%B6%E4%B8%8D%E6%98%AF%E4%B8%BA%E6%B0%91%E7%9D%80%E6%83%B3%EF%BC%8C%E7%9B%AE%E5%89%8D%E7%8E%AF%E5%A2%83%E5%A4%A7%E6%8A%B5%E5%A6%82%E6%AD%A4%EF%BC%8C%E6%B2%A1%E4%BB%80%E4%B9%88%E5%A5%BD%E6%8A%97%E4%BA%89%E7%9A%84

**Reply by richardo876** [reply]
> BE

%E8%A6%81%E6%8A%97%E4%BA%89%E3%80%82%E8%AE%A9%E5%85%B6%E4%BB%96%E4%BA%BA%E7%9F%A5%E9%81%93%E7%9C%9F%E5%AE%9E%E7%9A%84%E5%85%B1%E4%BA%A7%E5%85%9A%E5%B0%B1%E6%98%AF%E6%8A%97%E4%BA%89%E7%9A%84%E4%B8%80%E7%A7%8D%E6%96%B9%E5%BC%8F%E3%80%82%E5%A6%82%E5%B1%81%E5%90%90%E5%B1%81%E9%93%81%E9%93%BE%E5%A5%B3%E6%B2%B3%E5%8D%97%E5%82%A8%E6%88%B7%E5%92%8C%E8%BF%99%E6%AC%A1%E7%83%82%E5%B0%BE%E6%A5%BC%E4%BA%8B%E4%BB%B6%EF%BC%8C%E5%B0%B1%E6%98%AF%E5%BE%88%E5%A5%BD%E7%9A%84%E5%A5%91%E6%9C%BA%E8%AE%A9%E6%9B%B4%E5%A4%9A%E7%9A%84%E4%BA%BA%E7%9F%A5%E9%81%93%E7%9C%9F%E5%AE%9E%E7%9A%84%E5%85%B1%E4%BA%A7%E5%85%9A%E3%80%82

**Reply by KHW2099** [reply]
> %E5%8F%AA%E8%83%BD%E8%AF%B4%E5%BE%88%E9%9A%BE%EF%BC%8C%E5%9B%BD%E5%86%85%E6%B2%A1%E6%9C%89%E6%B8%B8%E8%A1%8C%E6%9D%83%E5%8A%9B%EF%BC%8C%E6%B2%A1%E6%9C%89%E5%8F%91%E8%A8%80%E6%9D%83%E5%8A%9B%EF%BC%8C%E4%BD%A0%E7%9C%8B%E7%9C%8B%E8%80%81%E5%A4%96%EF%BC%8C%E5%8A%A8%E4%B8%8D%E5%8A%A8%E5%B0%B1%E6%B8%B8%E8%A1%8C%E7%A4%BA%E5%A8%81%EF%BC%8C%E5%8A%A8%E4%B8%8D%E5%8A%A8%E5%B0%B1%E5%BE%80%E6%80%BB%E7%BB%9F%E8%BA%AB%E4%B8%8A%E6%89%94%E9%B8%A1%E8%9B%8B%EF%BC%8C%E6%81%B6%E6%90%9E%E6%80%BB%E7%BB%9F%E9%83%BD%E6%98%AF%E5%B0%8F%E4%BA%8B%E3%80%82%E5%9B%BD%E5%86%85%E5%B0%B1%E6%98%AF%E7%A5%9E%EF%BC%8C%E7%A2%B0%E4%B8%8D%E5%BE%97%E7%9A%84%E7%A5%9E%E3%80%82%0A%E8%BF%99%E7%A7%8D%E6%83%85%E5%86%B5%EF%BC%8C%E4%B8%8D%E6%98%AF%E4%B8%80%E4%B8%AA%E4%BA%BA%E8%83%BD%E6%89%AD%E8%BD%AC%E7%9A%84%EF%BC%8C%E9%99%A4%E9%9D%9E%E4%BD%A0%E6%98%AF%E5%9B%BD%E5%AE%B6%E9%A2%86%E5%AF%BC%E4%BA%BA%EF%BC%8C%E7%84%B6%E5%90%8E%E6%B4%BB%E5%BE%97%E4%BA%86%E6%9D%83%E5%8A%9B%EF%BC%8C%E7%84%B6%E5%90%8E%E4%BD%A0%E6%80%9D%E6%83%B3%E5%BC%80%E6%98%8E%EF%BC%8C%E8%A7%89%E5%BE%97%E5%BA%94%E8%AF%A5%E4%BA%BA%E4%BA%BA%E5%B9%B3%E7%AD%89%EF%BC%8C%E5%90%A6%E5%88%99%E6%97%A2%E5%BE%97%E5%88%A9%E7%9B%8A%E8%80%85%EF%BC%8C%E4%B8%8D%E4%BC%9A%E6%84%BF%E6%84%8F%E5%88%86%E4%BA%AB%E6%9D%83%E5%8A%9B%E7%BB%99%E4%BA%BA%E6%B0%91%EF%BC%8C%E4%BD%A0%E8%BF%9E%E4%BA%BA%E5%A4%A7%E4%BB%A3%E8%A1%A8%E6%98%AF%E6%80%8E%E4%B9%88%E9%80%89%E7%9A%84%E9%83%BD%E4%B8%8D%E7%9F%A5%E9%81%93%EF%BC%8C%E9%82%A3%E4%BA%9B%E4%BA%BA%E5%A4%A7%E4%BB%A3%E8%A1%A8%E8%BF%9E%E8%87%AA%E5%B7%B1%E6%98%AF%E6%80%8E%E4%B9%88%E8%A2%AB%E9%80%89%E4%B8%8A%E7%9A%84%E4%B9%9F%E4%B8%80%E6%97%A0%E6%89%80%E7%9F%A5%EF%BC%8C%E4%B8%80%E4%B8%AA%E4%B8%AA%E8%82%A5%E5%A4%B4%E5%A4%A7%E8%80%B3%EF%BC%8C%E8%AF%B4%E8%AF%9D%E5%86%A0%E5%86%95%E5%A0%82%E7%9A%87%EF%BC%8C%E5%8F%82%E5%8A%A0%E4%B8%A4%E4%BC%9A%EF%BC%8C%E6%97%A0%E9%9D%9E%E6%98%AF%E5%BD%B0%E6%98%BE%E8%87%AA%E5%B7%B1%E6%B4%BB%E5%BE%97%E6%9D%83%E5%8A%9B%E7%9A%84%E8%8D%A3%E8%80%80%EF%BC%8C%E5%B9%B6%E4%B8%8D%E6%98%AF%E4%B8%BA%E6%B0%91%E7%9D%80%E6%83%B3%EF%BC%8C%E7%9B%AE%E5%89%8D%E7%8E%AF%E5%A2%83%E5%A4%A7%E6%8A%B5%E5%A6%82%E6%AD%A4%EF%BC%8C%E6%B2%A1%E4%BB%80%E4%B9%88%E5%A5%BD%E6%8A%97%E4%BA%89%E7%9A%84

%E5%8E%86%E5%8F%B2%E5%91%A8%E6%9C%9F%E5%BE%88%E9%9A%BE%E6%89%93%E7%A0%B4%EF%BC%8C%E6%88%91%E4%BB%AC%E8%BF%99%E4%B8%80%E4%BB%A3%E6%88%96%E8%AE%B8%E5%8F%AA%E6%98%AF%E8%90%8C%E8%8A%BD%E9%98%B6%E6%AE%B5%EF%BC%8C%E5%8E%86%E5%8F%B2%E9%95%BF%E6%B2%B3%E4%B8%AD%E6%97%B6%E9%97%B4%E6%9C%80%E7%BB%88%E4%BC%9A%E7%BB%99%E5%87%BA%E7%AD%94%E6%A1%88%E3%80%82%E5%8F%AF%E6%83%9C%E7%9A%84%E6%98%AF%E9%80%9D%E5%8E%BB%E7%9A%84%E5%88%9D%E4%BB%A3%E7%BA%A2%E5%86%9B%EF%BC%8C%E9%93%81%E6%8B%B3%E6%B2%A1%E5%A4%9A%E4%B9%85%E5%8F%88%E9%94%A4%E5%90%91%E4%BB%96%E4%BB%AC%E7%9A%84%E5%90%8E%E4%BB%A3%E3%80%82

**Comment by HerSoul** [comment]
%E8%AF%B4%E8%AF%9D%E5%B0%8F%E5%BF%83%E7%82%B9%EF%BC%8C%E7%BA%A2%E7%A0%81%E5%8F%AF%E4%BB%A5%E6%97%A0%E7%97%9B%E5%8F%98%E7%81%B0%E5%85%A5%E7%9B%92

**Comment by cm-liang** [comment]
%E7%8E%B0%E5%9C%A8%E6%83%B3%E5%8F%91%E5%87%BA%E7%82%B9%E5%A3%B0%E9%9F%B3%E9%83%BD%E8%BF%99%E4%B9%88%E9%9A%BE%E4%BA%86%E5%90%97%EF%BC%9F

**Comment by richardo876** [comment]
%E5%93%88%E5%93%88%E6%89%80%E4%BB%A5%E8%BF%99%E6%A0%B7%E5%B0%B1%E5%8F%AF%E4%BB%A5%E8%81%8A%E4%BA%BA%E6%9D%83%E6%B0%91%E4%B8%BB%E8%87%AA%E7%94%B1%E4%BA%86%E5%90%97

**Reply by daxiazilong** [reply]
%E4%B8%8D%E7%AE%97%E5%8A%A0%E5%AF%86%E7%9A%84%E5%8A%A0%E5%AF%86%20%E5%BB%BA%E8%AE%AE%E8%AE%BA%E5%9D%9B%E8%BF%98%E6%98%AF%E5%85%B3%E4%BA%86%E5%90%A7%EF%BC%8C%E6%B2%A1%E5%95%A5%E7%94%A8%20%E9%99%A4%E4%BA%86%E6%8B%9B%E6%9D%A5%E4%B8%80%E5%B8%AE%E8%BD%AE%E5%AD%90%20%E5%92%8C%E8%8E%AB%E5%90%8D%E5%85%B6%E5%A6%99%E7%9A%84%E6%96%B0%E5%8F%B7%E7%9A%84%E4%BA%BA

**Reply by richardo876** [reply]
> 

%E5%85%B3%E4%BA%8E%E8%BD%AE%E5%AD%90%EF%BC%8C%E9%99%A4%E4%BA%86%E5%85%B1%E4%BA%A7%E5%85%9A%E5%91%8A%E8%AF%89%E4%BD%A0%E7%9A%84%E4%BF%A1%E6%81%AF%E4%BD%A0%E8%BF%98%E7%9F%A5%E9%81%93%E5%85%B6%E4%BB%96%E5%90%97%EF%BC%9F

**Comment by blisee** [comment]
这是什么呀

**Reply by CompetitiveLin** [reply]
URL编码

**Comment by daxiazilong** [comment, marked as answer]
110010101100011x100111010000110x101010000100111x1111111100001100x101000101101000x111001001111001x100111001001000x110011000101111x1000111101101110x101101101010000x101100x110010110110000x101001111110111x1111111100001100x111101101010110x101001111001101x111011010000100x1111111100001100x100000x111011100011111x100111011010110x101100110001000x111011010000100x1111111100001100x110101101111011x1000111101101110x101101101010000x1111111100001100x1000001101001001x101110000111100x111001110011011

**Reply by daxionga** [reply]
%E8%BF%99%E5%92%8B%E5%8F%88%E5%88%87utf-8%E4%BA%86

**Reply by nullxx-1** [reply]
> 
%E7%9C%8B%E6%9D%A5%E6%88%91%E8%A6%81%E4%B8%8A%E5%8A%A0%E5%AF%86%E4%BA%86

**Comment by 0xhelium** [comment]
编码完了发, 怕是有什么大病吧. 还这么多跟风的, 烂尾只能程序员关注???

**Comment by yanwuguangjun** [comment]
下面的很好玩？

**Comment by hsf25737** [comment]
加密对话是吧？

**Comment by kxbin** [comment]
6KaB5LiN5ZKx5Lus5p2l546p5Liq57yW56CB5ri45oiP77yM55yL55yL6K+45L2N5LiA5YWx6IO95Y+Y5o2i5aSa5bCR56eN57yW56CB77yf5bm25LiU55yL55yL5ZOq56eN57yW56CB6IO96KGo6L6+5pyA5aSa55qE5L+h5oGvYQ==

**Reply by kxbin** [reply]
--/---/.-./..././.-/.-../.--./..../.-/-..././-

**Reply by daxiazilong** [reply]
6L+Z5Liq5aSq5aW96Kej5p6Q5LqG5qC85byP5LiA55yL5bCx5piO55m95LqG77yM55m+5bqm5LiA5LiL5LiA5aCG6Kej5p6Q55qE5ZOI5ZOI

**Reply by kxbin** [reply]
濡傛灉涓嶈兘瑙ｆ瀽锛岄偅鎬庝箞鑳借浜鸿鎳傚憿锛?

**Reply by kxbin** [reply]
![1](https://user-images.githubusercontent.com/17782922/179648536-7a19bca4-9ee3-48fd-b9f5-bf2cd3e5c1d6.png)

**Comment by hktk1024** [comment]
5bCx5LiN5ZGK6K+J5L2gIQ==

**Reply by yida506** [reply]
bmltZW56YWlzaHVvc2hlbm1v

**Reply by iosyyy** [reply]
5ZWl57yW56CB6YO95pyJLi4

**Comment by apang1992** [comment]
55CG6K665LiK77yMZ2l0aHVi5piv5b6u6L2v55qE77yM55uW6Iyo5piv5Lit56eR6Zmi5aSW57GN6Zmi5aOr77yM5Lmf566X5piv5Lit5YWx55qE57uf5oiY5a+56LGh5LqG

**Comment by kxbin** [comment]
![1](https://user-images.githubusercontent.com/17782922/179648593-843ca9fa-d5e6-42a7-b76a-4121757dffe0.png)

**Reply by iosyyy** [reply]
![image](https://user-images.githubusercontent.com/55818117/179649508-c201f9fe-c0e0-4433-982a-1bbec48b9aaa.png)

**Reply by kxbin** [reply]
![2](https://user-images.githubusercontent.com/17782922/179651028-d64951d7-d04a-4096-97b9-8e27be538096.png)
